### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by LOctavio**
### For this branch, I did the following

- Create a table named owners with the following columns:
  - id: integer (set it as autoincremented PRIMARY KEY)
  - full_name: string
  - age: integer
- Create a table named species with the following columns:
  - id: integer (set it as autoincremented PRIMARY KEY)
  - name: string
- Modify animals table
  - Remove column species
  - Add column species_id which is a foreign key referencing species table
  - Add column owner_id which is a foreign key referencing the owners table
- Insert the following data into the owners table:
  - Sam Smith 34 years old.
  - Jennifer Orwell 19 years old.
  - Bob 45 years old.
  - Melody Pond 77 years old.
  - Dean Winchester 14 years old.
  - Jodie Whittaker 38 years old.
- Insert the following data into the species table:
  - Pokemon
  - Digimon
- Modify animals so it includes the species_id value:
  - If the name ends in "mon" it will be Digimon
  - All other animals are Pokemon
- Modify animals to include owner information (owner_id):
  - Sam Smith owns Agumon.
  - Jennifer Orwell owns Gabumon and Pikachu.
  - Bob owns Devimon and Plantmon.
  - Melody Pond owns Charmander, Squirtle, and Blossom.
  - Dean Winchester owns Angemon and Boarmon.
- Write queries (using JOIN) to answer the following questions:
  - What animals belong to Melody Pond?
  - List of all animals that are pokemon (their type is Pokemon).
  - List all owners and their animals, remember to include those that don't own any animal.
  - How many animals are there per species?
  - List all Digimon owned by Jennifer Orwell.
  - List all animals owned by Dean Winchester that haven't tried to escape.
  - Who owns the most animals?

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -19,4 +19,35 @@ values ('Angemon', DATE '2005-06-12', 1, true, -45);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
 values ('Boarmon', DATE '2005-06-07', 7, true, 20.4);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
-values ('Blossom', DATE '1998-10-13', 3, true, 17);
\ No newline at end of file
+values ('Blossom', DATE '1998-10-13', 3, true, 17);
```
Comment (Emmyn5600) [inline data.sql]: - [ ] You are missing to insert the data in the species table
- [ ] Let us keep plural form  our query solution (`INSERT, INTO and VALUES`)

File: data.sql [hunk 2]
```diff
@@ -19,4 +19,35 @@ values ('Angemon', DATE '2005-06-12', 1, true, -45);
...
+values ('Melody Pond', 77);
+INSERT INTO owners(full_name, age)
+values ('Dean Winchester', 14);
+INSERT INTO owners(full_name, age)
+values ('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+values ('Pokemon');
+INSERT INTO species(name)
+values ('Digimon');
```
Comment (Emmyn5600) [inline data.sql]: - [ ] Let us keep plural form  our query solution (`INSERT, INTO and VALUES`)

File: data.sql [hunk 3]
```diff
@@ -19,4 +19,35 @@ values ('Angemon', DATE '2005-06-12', 1, true, -45);
...
+
+BEGIN;
+UPDATE animals SET species_id = 2 WHERE name like '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id is null;
```
Comment (Emmyn5600) [inline data.sql]: - [ ] Instead of setting the value using IDs, let us use select id from species that is match with condition.

File: data.sql [hunk 4]
```diff
@@ -19,4 +19,35 @@ values ('Angemon', DATE '2005-06-12', 1, true, -45);
...
+COMMIT;
+
+BEGIN;
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' or name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' or name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' or name = 'Squirtle' or name = 'Blossom';
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' or name = 'Boarmon';
+COMMIT;
```
Comment (Emmyn5600) [inline data.sql]: - [ ]Again,  Instead of setting the value using IDs, let us use select id from owners that is match with condition.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @LOctavio 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

  Changes required :recycle: 
    
      Please check the comment under the review too


    Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**